src/pages/graphql-bootcamp.js

```diff
@@ -56,16 +56,16 @@ const GraphQL = () => (
     />
     <TopSection>
       <Grid>
-        <CallToActionRow left>
+        {/* <CallToActionRow left>
```

Comment:
@shakersinthedark we should try not to leave commented code in the project. It's like having a tidy house :). Can you please remove the comment or add a note about why we have that commented code there?